TestResultSummaryService/parsers/BenchmarkMetric.js

```diff
@@ -48,6 +48,15 @@ const BenchmarkMetricRegex = {
                 regexRepeat: false
             }
         ]
+    },
+    AcmeAir_Nodejs: {
+        metrics: [
+            {
+                name: "Global Throughput",
```

Comment:
@jiayaolinn As discussed, `Throughput` instead of `Global Throughput` would be more applicable for AcmeAir.